admin/controllers/check-form.js

```diff
@@ -172,6 +172,18 @@ const saveCheckForm = async (req, res, next) => {
     })
   }
 
+  try {
+    const fileName = await checkFormService.buildFormName(uploadFile.filename)
+    if (!fileName) {
+      req.flash('error', `${fileName} already exists. Rename and upload again.`)
+      return res.redirect('/test-developer/upload-new-form')
+    } else {
```

Comment:
We don't need the else part here... can just use the return in the `if` part above a a guard.  This simplifies and clarifies the intent. E.g.

```
if (!fileName) {
      req.flash('error', `${fileName} already exists. Rename and upload again.`)
      return res.redirect('/test-developer/upload-new-form')
} 
checkForm.name = fileName
```